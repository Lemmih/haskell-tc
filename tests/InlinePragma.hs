module InlinePragma where

{-# NOINLINE x #-}
x () = ()

{-# INLINE y #-}
y () = ()
