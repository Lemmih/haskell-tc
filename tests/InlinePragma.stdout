Bindings:
:: () â†’  ()
  [4mx () = ()[0m
:: () â†’  ()
  [4my () = ()[0m

Proofs:
coercion: ()
  x [4m()[0m = ()
coercion: ()
  y [4m()[0m = ()
